Description: Use program default instead of setting paths in config file
Author: Petter Reinholdtsen <pere@debian.org>

Forwarded: no
Reviewed-By: Petter Reinholdtsen <pere@debian.org>
Last-Update: 2013-11-02

--- plinth-0.3.0.0.git.20131101.orig/plinth.sample.config
+++ plinth-0.3.0.0.git.20131101/plinth.sample.config
@@ -3,16 +3,6 @@ product_name = Plinth
 box_name = FreedomBox
 
 [Path]
-file_root = %(root)s
-python_root = %(root)s
-data_dir = %(file_root)s/data
-store_file = %(data_dir)s/store.sqlite3
-user_db = %(data_dir)s/users
-status_log_file = %(data_dir)s/status.log
-access_log_file = %(data_dir)s/access.log
-users_dir = %(data_dir)s/users
-pidfile = %(data_dir)s/pidfile.pid
-server_dir = plinth/
 
 [Network]
 host = 127.0.0.1
